README.md

```diff
@@ -15,16 +15,20 @@ Then install the framework into the `lib/` directory using:
 
 ```
 mkdir lib/
-pip install -t lib/ https://github.com/canonical/operator
+pip3 install -t lib/ https://github.com/canonical/operator/archive/master.zip
 ```
 
+> Note: Due to [pip#3826](https://github.com/pypa/pip/issues/3826), you may get
+> a "can't combine user with prefix" if you are using pip3 provided by Ubuntu
+> prior to 19.04, in which case you simply need to add `--system` to the command.
+
 Your `src/charm.py` is the entry point for your charm logic. At a minimum, it
```

Comment:
Why not `src/lib/charm.py`? If that was the case, any hooks/<symlink> -> ../lib/charm.py execution would result in the lib directory being added as the first entry in `sys.path`:

```
sys.path
['/home/ubuntu/src/charm-test/lib', ...
```

Which would remove the need for `sys.path.append('lib')`.

https://docs.python.org/3/tutorial/modules.html#the-module-search-path
```
sys.path is initialized from these locations:
* The directory containing the input script (or the current directory when no file is specified).
...
On file systems which support symlinks, the directory containing the input script is calculated after the symlink is followed. In other words the directory containing the symlink is not added to the module search path.
```

Reply:
My comment above was answered during our call and can be ignored.